### DISCUSSION TITLE
Probleme mise à jour 1.12.2

### DISCUSSION THREAD

**Discussion by Jelogofr** [Domotrique/okovision_2023#11] [Q&A] [answered]
Bonjour Domotrique,

     je viens de réaliser la mise à jour via Okovision. Et dans la page d’accueil, je n'ai plus les valeurs de conso et de température. Elles sont toutes à 0 et j'ai ce message d'erreur en rouge "Problème de communication : rendu.getIndicByDay"

Que dois je faire pour faire régler le problème?

En te remerciant, et félicitation pour avoir réussi à mettre la fonction backup.

**Comment by Domotrique** [comment]
Salut @Jelogofr,

Il faut peut-etre que tu fasses un CTRL+F5 sur ta page pour bien mettre à jour le cache.

Si le problème est toujours là, envoie moi un screen si possible. Merci

**Reply by Jelogofr** [reply]
Re @Domotrique,

Malgré le rafraichissement, j'ai cette erreur que tu verras sur la capture dont je te joins.

Je te remercie pour l'aide apporter.

![Capture d’écran 2024-02-08 132311](https://github.com/Domotrique/okovision_2023/assets/155993718/2221a694-85d5-4e79-b4a3-31a930999d4d)

**Reply by Domotrique** [reply]
Re @Jelogofr 
Alors j'ai effectivement fait une modif mineure dans la fonction où ça crash.
Tu as combien de circuits sur ta chaudière ?

**Reply by Jelogofr** [reply]
@Domotrique 
J'en ai deux pour le chauffage et un pour les ballon d'eau chaude (ECS)

**Reply by Domotrique** [reply]
Tu peux m'envoyer ton plus récent fichier csv produit par ta chaudière stp ?

**Reply by Jelogofr** [reply]
[touch_20240208.csv](https://github.com/Domotrique/okovision_2023/files/14209342/touch_20240208.csv)

**Reply by Domotrique** [reply, marked as answer]
En fait c'est pas nécessaire, je pense avoir trouvé le soucis.
J'avais corrigé la façon dont était lu un paramètre renvoyé par une autre fonction (celle qui renvoyait la conso liée à l'ECS) car cela me générait des erreurs un peu similaires à ce que tu as eu.

En fait il fallait corriger la façon dont cette autre fonction renvoyait le paramètre dans le cas où il n'y avait pas de circuit d'ECS (ce qui est mon cas).

Je vais envoyer un fix et je t'invite à forcer la nouvelle maj en changeant le numéro de version dans le fichier /_include/version.json en passant de 1.12.2 à 1.12.1, ensuite tu pourras refaire la meme maj que tu as fait plus tôt.

**Reply by Jelogofr** [reply]
@Domotrique 

C'est bon avec la nouvelle version 1.12.4. tout est revenu à la normal!
Bravo et merci pour le soutien.
Bonne continuation.
